tests/e2e/env/setup.sh

```diff
@@ -67,27 +84,29 @@ if [[ "$E2E_USE_LOCAL_SERVER" != false ]]; then
 	echo "Secrets created"
 
 	step "Starting SERVER containers"
-	redirect_output docker compose -f docker-compose.yml -f docker-compose.e2e.yml up --build --force-recreate -d
+	redirect_output docker compose -p transact-platform-server-e2e -f docker-compose.yml -f docker-compose.e2e.yml up --build --force-recreate -d
```

Comment:
Specifying the project name is not required. Instead, we can rename the dependency directory for `transact-platform-server` to `transact-platform-server-e2e`. Same for dec tools directory.

Since docker picks up the immediate parent directory as project name, `transact-platform-server-e2e` will be picked as the name automatically and we don't need to specify it everywhere. Also we don't need to specify it using the `COMPOSE_PROJECT_NAME` env variable.

Reply:
List of changes to be done in this approach.

On `env/shared.sh`:
* Replace `export SERVER_PATH="$E2E_ROOT/deps/transact-platform-server"` with `export SERVER_PATH="$E2E_ROOT/deps/transact-platform-server-e2e"`
* Replace `export DEV_TOOLS_DIR="wcp-dev-tools"` with `export DEV_TOOLS_DIR="wcp-dev-tools-e2e"`

Reply:
Implemented this great suggestion in 81cfb994a